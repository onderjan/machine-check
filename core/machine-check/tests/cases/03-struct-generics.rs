#[machine_check_macros::machine_description]
mod machine {
    struct A<G> {}
}

fn main() {}
