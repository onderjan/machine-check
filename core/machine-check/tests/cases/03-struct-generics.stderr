error: machine-check: Generics not supported
 --> tests/cases/03-struct-generics.rs:3:13
  |
3 |     struct A<G> {}
  |             ^^^
